src/settings-ui/Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -56,6 +56,11 @@
                                     x:Uid="PowerLauncher_IgnoreHotkeysInFullScreen"
                                     IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.IgnoreHotkeysInFullScreen}" />
                             </labs:SettingsCard>
+                            <labs:SettingsCard ContentAlignment="Left">
```

Comment:
The placement of the new setting can be improved. The new setting gas nothing to do with the activation shortcut.

I think it should be placed in the results settings (max utem count, ...) group.

Reply:
@htcfreek I have moved it, see newest commit